dashboard/legacy/middleware/net_sim_api.rb

```diff
@@ -549,7 +549,9 @@ def parse_table_map_from_query_string(query_string)
 def parse_ids_from_query_string(query_string)
   [].tap do |ids|
     CGI.parse(query_string)['id[]'].each do |id|
-      ids << Integer(id, 10) rescue ArgumentError
```

Comment:
I have to say, this looks like a potential misuse of the "modifier" syntax -- it looks like the caller wanted to rescue this type of error, rather than insert this error class into the list of returned ids. in fact it looks like this error was introduced specifically when switching to the modifier syntax: https://github.com/code-dot-org/code-dot-org/commit/d04d4120d30b6220332a616ccec430b68dbdd72f

Reply:
Oh, great catch! Definitely agree with that analysis; I'm tempted to manually update this fix to reflect that presumed intent rather than the current functionality.